comid-triples.cddl

```diff
@@ -0,0 +1,178 @@
+concise-mid-tag = {
+  ; TODO(tho) language
+  ; TODO(tho) tag-id
+  ; TODO(tho) module-name
+  ; TODO(tho) entity
```

Comment:
Previously this was named 'module-entity-map' and contained 'entity-name', 'reg-id' and 'role' and was extensible.